### PR TITLE
Patch: improve performance - new secondary keys

### PR DESCRIPTION
**Pull request by christianguenter2**
Should give considerably better patch performance on large diffs.

### CODE REVIEW

File: src/utils/zcl_abapgit_diff.clas.abap [hunk 1]
```diff
@@ -351,16 +345,9 @@ CLASS zcl_abapgit_diff IMPLEMENTATION.
     DATA: lv_old_num TYPE i.
```
Comment (mbtools) [inline src/utils/zcl_abapgit_diff.clas.abap:345]: Remove lv_old_num

File: src/utils/zcl_abapgit_diff.clas.abap [hunk 2]
```diff
@@ -327,16 +328,9 @@ CLASS zcl_abapgit_diff IMPLEMENTATION.
     DATA: lv_new_num TYPE i.
```
Comment (mbtools) [inline src/utils/zcl_abapgit_diff.clas.abap:328]: Remove lv_new_num

**Review by mbtools — CHANGES_REQUESTED** [review]
delete unused vars

File: src/utils/zcl_abapgit_diff.clas.abap (hunk 1 above)
Comment (christianguenter2) [reply src/utils/zcl_abapgit_diff.clas.abap]: done

File: src/utils/zcl_abapgit_diff.clas.abap (hunk 2 above)
Comment (christianguenter2) [reply src/utils/zcl_abapgit_diff.clas.abap]: done

**Review by mbtools — APPROVED** [review]
👍

**Pull request merged**